ci/setup_env.sh

```diff
@@ -103,7 +103,7 @@ conda list pandas
 # Make sure any error below is reported as such
 
 echo "[Build extensions]"
-python setup.py build_ext -q -j2
+python setup.py build_ext -q -j2 --with-cython-coverage
```

Comment:
im pretty sure we dont want this on every build, since it slows things down a lot.

Reply:
makes sense.. would you know why the build is failing though?

Reply:
pandas/_libs/algos.c: In function ‘__pyx_fuse_0__pyx_f_6pandas_5_libs_5algos_rank_sorted_1d.isra.0.constprop’:
pandas/_libs/algos.c:1023:34: error: ‘__pyx_gilstate_save’ may be used uninitialized in this function [-Werror=maybe-uninitialized]
 1023 | #define __Pyx_PyGILState_Release PyGILState_Release
      |                                  ^~~~~~~~~~~~~~~~~~
pandas/_libs/algos.c:73649:20: note: ‘__pyx_gilstate_save’ was declared here
73649 |   PyGILState_STATE __pyx_gilstate_save;